features/atomic/docker_pull.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Docker pull
+  Run 'docker pull' on list of images
+
+  Scenario Outline: docker pull
+      Given "all" hosts from dynamic inventory
+       Then docker pull "<image>:<tag>"
+
+  Examples: docker images
+    | image     | tag    |
```

Comment:
Formatting slightly off here when viewing via GitHub.  Might be correct in a checkout, though.